### PR TITLE
Documentations update

### PR DESCRIPTION
**Pull request by amrasekh**
Added and updated docstring, rst, and notebook documentations

### CODE REVIEW

File: docs/tutorials.rst [hunk 1]
```diff
@@ -7,12 +7,52 @@ Tutorials
...
    notebooks/integration_demo
+   notebooks/lens_customization
 
 
 .. toctree::
    :caption: Demo Use Cases
    :maxdepth: 1
 
    notebooks/lens_demos/binaryclassification
+   docs/notebooks/module_demos/fairness_nlp
```
Comment (IanAtCredo) [inline docs/tutorials.rst:19]: this should be `notebooks/module_demos...` not `docs/notebooks/module_demos...`

However, I don't think we should put `module_demos` here. We should have another section for them. Using modules directly should be hidden as much as possible (that's the current working hypothesis anyway). I want to direct people to use lens.

File: credoai/assessment/nlp_utils.py [hunk 2]
```diff
@@ -11,6 +23,14 @@ def gpt2_text_generator(prompt):
     return response
 
 def get_default_nlp_assessments():
+    """Loads the default nlp assessment model
+    The model assesses a text for toxicity
```
Comment (IanAtCredo) [inline credoai/assessment/nlp_utils.py:27]: missing new line

**Review by IanAtCredo — APPROVED** [review]
Minor changes requested before you merge.

File: credoai/assessment/nlp_utils.py (hunk 2 above)
Comment (amrasekh) [reply credoai/assessment/nlp_utils.py]: added now

File: docs/tutorials.rst (hunk 1 above)
Comment (amrasekh) [reply docs/tutorials.rst]: Agreed. Need to please both people who are more into exploring repos and others who prefer readthedocs.

Comment (amrasekh) [reply docs/tutorials.rst]: link fixed

**Pull request merged**